Formula/pigz.rb

```diff
@@ -39,7 +39,7 @@ def install
     assert (testpath/"example.gz").file?
     system bin/"unpigz", testpath/"example.gz"
     assert_equal test_data, (testpath/"example").read
-    system "/bin/dd", "if=/dev/random", "of=foo.bin", "bs=1m", "count=10"
+    system "/bin/dd", "if=/dev/random", "of=foo.bin", "bs=1024k", "count=10"
```

Comment:
In linuxbrew-core, we fixed this by changing `1m` to `1M`, which is also valid.  Can we use that instead?  That will be less work as we try to zero the diffs between the repos.

Reply:
That was what I tried first.
Sadly, `M` is Linux-specific and not valid on macOS/BSD.
On other hand, `m` is BSD-specific.
`k` is POSIX compliant.

Reply:
Ah I guess I never had a chance to try that on macOS.  We'll just have to use `1024k`.  Once we merge this I'll have to open a PR in linuxbrew-core to align this but it will be trivial.